beacon-chain/blockchain/receive_block.go

```diff
@@ -73,6 +73,10 @@ func (s *Service) ReceiveBlock(ctx context.Context, block interfaces.SignedBeaco
 	if err := logBlockSyncStatus(blockCopy.Block(), blockRoot, justified, finalized, receivedTime, uint64(s.genesisTime.Unix())); err != nil {
 		return err
 	}
+	// Log payload data
+	if err := logPayload(blockCopy.Block()); err != nil {
+		return err
```

Comment:
```suggestion
		log.WithError(err).Error("Unable to log debug block payload data")
```

I think we should move away from this pattern of failing to process a block if logs fail for some reason. 

Perhaps you could change this pattern for all of these log statements that return an error. `logBlockSyncStatus` & `logStateTransitionData`

Reply:
@prestonvanloon great suggestion, I modified the other two log statements.